ui/Rhino/AGS/dev/AGS_form_from_json_cmd.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+import rhinoscriptsyntax as rs
+import compas_rhino
+
+from compas_ags.diagrams import FormGraph
+from compas_ags.diagrams import FormDiagram
+from compas_ags.diagrams import ForceDiagram
+
+from compas.geometry import Line
+from compas.geometry import is_point_on_line
+from compas_rhino.utilities import get_object_layers
+
+__commandname__ = "AGS_form_from_json"
```

Comment:
It is indeed confusing. Should this be removed?

If not, I can remove the graph generation and graph checking, in such way it can be useful for a user that comes with a json from somewhere else?

Reply:
i would reserve the json stuff for sessions only. otherwise it will be unclear what is contained in the json files... so yes, remove!